volcano plot

naqvia opened the pull request:
Suppl volcano plot comparing differentially low (N=12) Exon 4 PSI samples vs high. The purpose of these plots and PR is to show that due to increased heterogeneity of Exon 4 splicing we could harness that to elucidate Exon 4 splicing mediated impact.

jharenza reviewed (commented):
You have 3 08* scripts in here:
08-plot-Ex4-PSI-all-tumors.R
08-plot-Ex4PSI-correlations.R
08-volcano-plot-high-vs-low-Ex4-PSI.R

Can you organize and update the run script please?

jharenza commented on analyses/CLK1-splicing_correlations/08-plot-Ex4PSI-correlations.R:
```diff
@@ -0,0 +1,216 @@
+################################################################################
+# 08-plot-Ex4PSI-correlations.R
```
Some errors in this script:
>Error in `purrr::map2()`:
ℹ In index: 1.
ℹ With name: hgg_bs_id.
Caused by error in `library()`:
! there is no package called ‘clusterProfiler’
Backtrace:
     ▆
  1. ├─purrr::map2(sample_id_groups, output_names, process_rmats_data)
  2. │ └─purrr:::map2_("list", .x, .y, .f, ..., .progress = .progress)
  3. │   ├─purrr:::with_indexed_errors(...)
  4. │   │ └─base::withCallingHandlers(...)
  5. │   ├─purrr:::call_with_cleanup(...)
  6. │   └─global .f(.x[[i]], .y[[i]], ...)
  7. │     └─base::library("clusterProfiler")
  8. │       └─base::stop(packageNotFoundError(package, lib.loc, sys.call()))
  9. └─purrr (local) `<fn>`(`<pckgNtFE>`)
 10.   └─cli::cli_abort(...)
 11.     └─rlang::abort(...)
Warning message:
In DESeqDataSet(se, design = design, ignoreRank) :
  some variables in design formula are characters, converting to factors
Execution halted

naqvia commented:
So, I deleted erroneous scripts and added command to the run_script. For some reason ClusterProfiler installation has broken in Docker installation step. Im not sure why but I am investigating. I use this for all of ORA so its critical that we figure this out. For now, you can look at the volcano plots and run the module... @jharenza

naqvia commented:
I also fixed the Dockerfile for clusterProfile and library related to PR #22 . Both PRs should run fine now! @jharenza
